### ISSUE TITLE
dead links in docs

### ISSUE THREAD

**Issue by tacaswell** [matplotlib/matplotlib#6678]
While prepping for 1.5.2 I ran linkchecker against external targets, this is the list of things that I could not easily resolve

```
URL        `http://aspnetresources.com/tools/colorBlindness'
Name       `http://aspnetresources.com/tools/colorBlindness'
Parent URL file:///home/tcaswell/source/p/matplotlib/doc/build/html/users/colormaps.html, line 408, col 86
Real URL   http://www.aspnetresources.com/tools/colorBlindness
Check time 3.353 seconds
Info       Redirected to
           `http://www.aspnetresources.com/tools/colorBlindness'.
Result     Error: 404 Not Found

URL        `http://aspnetresources.com/tools/colorBlindness'
Name       `http://aspnetresources.com/tools/colorBlindness'
Parent URL file:///home/tcaswell/source/p/matplotlib/doc/build/html/users/colormaps.html, line 414, col 75
Real URL   http://www.aspnetresources.com/tools/colorBlindness
Check time 3.336 seconds
Info       Redirected to
           `http://www.aspnetresources.com/tools/colorBlindness'.
Result     Error: 404 Not Found
```

This site seems to be taken over by a Japanese website that google translate tells me is selling health insurance 

---

RESOLVED

```
URL        `http://mindtrove.info/articles/python-weak-references/'
Name       `“Mindtrove” blog'
Parent URL file:///home/tcaswell/source/p/matplotlib/doc/build/html/api/cbook_api.html, line 211, col 1
Real URL   http://mindtrove.info/articles/python-weak-references/
Check time 0.196 seconds
Result     Error: 404 Not Found

```

Contacted author

---

RESOLVED

```
URL        `http://www.jwave.vt.edu/%7Erkriz/Projects/create_color_table/color_07.pdf'
Name       `Rainbow Color\nMap (Still) Considered Harmful'
Parent URL file:///home/tcaswell/source/p/matplotlib/doc/build/html/api/pyplot_summary.html, line 954, col 58
Real URL   http://www.jwave.vt.edu/~rkriz/Projects/create_color_table/color_07.pdf
Check time 0.113 seconds
Result     Error: ConnectionError: HTTPConnectionPool(host='www.jwave.vt.edu', port=80): Max retries exceeded with url: /~rkriz/Projects/create_color_table/color_07.pdf (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection objec...
```

Entire domain is gone

---

RESOLVED

```
URL        `http://mpl.code.astraw.com/overview.html'
Name       `image comparison'
Parent URL file:///home/tcaswell/source/p/matplotlib/doc/build/html/users/whats_new.html, line 2339, col 16
Real URL   http://mpl.code.astraw.com/overview.html
Check time 0.424 seconds

```

```
URL        `http://mpl-buildbot.code.astraw.com/'
Name       `buildbot'
Parent URL file:///home/tcaswell/source/p/matplotlib/doc/build/html/users/credits.html, line 154, col 41
Real URL   http://mpl-buildbot.code.astraw.com/
Check time 0.090 seconds

```

domains seem to be gone attn @astraw

**Comment by tacaswell** [comment]
mindtrove link is fixed, jwave link is replaced to pay-walled version of paper in #6679

**Comment by astraw** [comment]
I think in both cases the links to the astraw.com domain are only minimally important and it would be difficult for me to resurrect them, so I propose just to remove those links from the docs.

**Comment by tacaswell** [comment]
Thanks!

**Comment by QuLogic** [comment]
The first one seems dead; maybe can be replaced by this one: http://www.color-blindness.com/coblis-color-blindness-simulator/

**Comment by QuLogic** [comment]
At this point, searching for aspnetresources.net just points to this page and the dead one; I think there's little hope of it coming back.

**Comment by NelleV** [comment]
I think this is fixed.
@tacaswell can you confirm and close if that is the case?

```shell
nelle@tamaris> linkchecker .               ~/Projects/matplotlib/doc/build/html
INFO 2016-11-30 13:13:10,925 MainThread Checking intern URLs only; use --check-extern to check extern URLs.
LinkChecker 9.3              Copyright (C) 2000-2014 Bastian Kleineidam
LinkChecker comes with ABSOLUTELY NO WARRANTY!
This is free software, and you are welcome to redistribute it
under certain conditions. Look at the file `LICENSE' within this
distribution.
Get the newest version at http://wummel.github.io/linkchecker/
Write comments and bugs to https://github.com/wummel/linkchecker/issues
Support this project at http://wummel.github.io/linkchecker/donations.html

Start checking at 2016-11-30 13:13:10-007
 9 threads active,   713 links queued,  482 links in  25 URLs checked, runtime 1 seconds
 9 threads active,  3253 links queued, 2955 links in  71 URLs checked, runtime 6 seconds
 9 threads active,  6366 links queued, 5045 links in 113 URLs checked, runtime 11 seconds
10 threads active,  8179 links queued, 7957 links in 135 URLs checked, runtime 16 seconds
 7 threads active,  9419 links queued, 10874 links in 151 URLs checked, runtime 21 seconds
 9 threads active,  9210 links queued, 15444 links in 309 URLs checked, runtime 26 seconds
 7 threads active, 10120 links queued, 19788 links in 627 URLs checked, runtime 31 seconds
10 threads active, 10673 links queued, 23940 links in 806 URLs checked, runtime 36 seconds
10 threads active, 10926 links queued, 29252 links in 1065 URLs checked, runtime 41 seconds
10 threads active, 11294 links queued, 33587 links in 1157 URLs checked, runtime 46 seconds
 8 threads active, 11087 links queued, 38785 links in 1367 URLs checked, runtime 51 seconds
 7 threads active, 11063 links queued, 43751 links in 1673 URLs checked, runtime 56 seconds
 9 threads active, 11091 links queued, 49035 links in 2049 URLs checked, runtime 1 minute, 1 seconds
 9 threads active, 11709 links queued, 53179 links in 2307 URLs checked, runtime 1 minute, 6 seconds
 8 threads active, 11867 links queued, 57493 links in 2527 URLs checked, runtime 1 minute, 11 seconds
 9 threads active, 11941 links queued, 61715 links in 2751 URLs checked, runtime 1 minute, 16 seconds
 8 threads active, 12196 links queued, 65166 links in 2978 URLs checked, runtime 1 minute, 21 seconds
 9 threads active, 12381 links queued, 69307 links in 3167 URLs checked, runtime 1 minute, 26 seconds
10 threads active, 12291 links queued, 73905 links in 3386 URLs checked, runtime 1 minute, 31 seconds
 9 threads active, 12361 links queued, 78225 links in 3556 URLs checked, runtime 1 minute, 36 seconds
 8 threads active, 12620 links queued, 82365 links in 3733 URLs checked, runtime 1 minute, 41 seconds
10 threads active, 12677 links queued, 86293 links in 3966 URLs checked, runtime 1 minute, 46 seconds
 7 threads active, 12414 links queued, 90670 links in 4247 URLs checked, runtime 1 minute, 51 seconds
 4 threads active,  6311 links queued, 101102 links in 4551 URLs checked, runtime 1 minute, 56 seconds
 7 threads active,  4311 links queued, 108154 links in 4932 URLs checked, runtime 2 minutes, 1 seconds
 9 threads active,  3732 links queued, 112588 links in 6628 URLs checked, runtime 2 minutes, 6 seconds
 2 threads active,     1 link queued, 119402 links in 8155 URLs checked, runtime 2 minutes, 11 seconds

Statistics:
Downloaded: 37.6MB.
Content types: 9429 image, 86045 text, 0 video, 0 audio, 9324 application, 3 mail and 14701 other.
URL lengths: min=14, max=156, avg=78.

That's it. 119502 links in 8155 URLs checked. 0 warnings found. 0 errors found.
Stopped checking at 2016-11-30 13:15:22-007 (2 minutes, 11 seconds)
```

**Comment by jenshnielsen** [comment]
This isse is about external links. You have to pass --check-extern to link checker to check those. The internal ones are checked by Travis automatically

**Comment by NelleV** [comment]
OK.

**Comment by NelleV** [comment]
closed in #7539
